### PR TITLE
monitoring: refactor prometheus scrape config generation

### PR DESCRIPTION
**Pull request by WilliButz**
- removes implicit SSH TCP blackbox checks

### CODE REVIEW

File: monitoring/default.nix [hunk 1]
```diff
@@ -150,82 +189,39 @@ in {
...
-                { source_labels = [ "__param_target" ];
-                  regex = "(.*)";
-                  target_label = "instance";
-                  replacement = ''''${1}'';
-                }
-                {
-                  source_labels = [];
-                  regex = ".*";
-                  target_label = "__address__";
-                  replacement = "127.0.0.1:9115";
```
Comment (fpletz) [inline monitoring/default.nix]: Alarm! All prometheus instances always queried the local blackbox exporter! \o/

Comment (fpletz) [reply monitoring/default.nix]: According to the code the idea seemed to be to query the blackbox exporters of the other prometheus instances. The metrics were tagged that way but instead the local blackbox exporter was always queried. Now we explicitely use the local blackbox exporter but not three times… ;)

File: monitoring/default.nix [hunk 2]
```diff
@@ -5,31 +5,28 @@ with lib;
 let
   cfg = config.mayflower.monitoring;
   hostName = name: machine:
-    machine.deployment.targetHost or 
+    machine.deployment.targetHost or
       "${name}.${machine.containerDomains.${machine.hostBridge}}";
   hostNames = hosts: mapAttrsToList hostName hosts;
-  # machine config attrs -> { containerName = container machine config // hostBridge // containerDomains }
+  #" machine config attrs -> { containerName = container machine config // hostBridge // containerDomains }
```
Comment (globin) [inline monitoring/default.nix]: 🤔

Comment (fpletz) [reply monitoring/default.nix]: vim syntax highlighting fix :hankey:

**Pull request merged**